caldate_fmjd.c

```diff
@@ -0,0 +1,48 @@
+#include "caldate.h"
+
+void caldate_frommjd(cd,day,pwday,pyday)
+struct caldate *cd;
+long day;
+int *pwday;
+int *pyday;
+{
+  long year;
+  long month;
+  int yday;
+
+  year = day / 146097L;
```

Comment:
I bet this is correct, but I have no clue what's going on here with all those magic numbers.

Reply:
just writing ```(365 * 400)``` would be a start, as every existing compiler will just do the multiplication at compile time anyway, but a comment what this was all about (if someone knows) is even better.